### ISSUE TITLE
Question about proposals matching

### ISSUE THREAD

**Issue by cocoshe** [ShoufaChen/DiffusionDet#79]
According to Hungarian algorithm, we can get the bipartite graph maximum matching between ground truth bounding boxes and proposals, that means we can find a proposal for each gt?

If so, it seems to only have positive samples because when we calculate the box loss, we use L1 loss between the matched box pairs, how can we train with neg samples? And if this situation may occur: there are **two(more than one)** proposals and they are both similar to **one** gt box, we only choose the one of them with the highest class logits? Though the others are also with the high class logits, but dropped without being pos samples?
